src/util/Permissions.js

```diff
@@ -32,12 +32,26 @@ class Permissions {
 
   /**
    * Gets all given permissions that are missing from the bitfield.
-   * @param {PermissionResolvable[]} permissions Permissions to check for
+   * @param {PermissionResolvable|PermissionResolvable[]} permissions Permission(s) to check for
    * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override
    * @returns {PermissionResolvable[]}
    */
   missing(permissions, checkAdmin = true) {
-    return permissions.filter(p => !this.has(p, checkAdmin));
+    if (permissions instanceof Array) return permissions.filter(p => !this.has(p, checkAdmin));
+    return this.missing(new Permissions(permissions).toArray(checkAdmin), checkAdmin);
+  }
+
+  /**
+   * Gets an {@link Array} of permission names (such as VIEW_CHANNEL) based on the permissions available.
+   * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override
+   * @returns {PermissionResolvable[]}
+   */
+  toArray(checkAdmin = true) {
```

Comment:
This can simply return `Object.keys(this.constructor.FLAGS).filter(perm => this.has(perm, checkAdmin))`